test/kitchen/kitchen-vagrant-security-agent.yml

```diff
@@ -0,0 +1,28 @@
+---
+driver:
+  name: vagrant
+  <% if ENV['KITCHEN_VAGRANT_PROVIDER'] %>
+  provider: <%= ENV['KITCHEN_VAGRANT_PROVIDER'] %>
+  <% end %>
+
+provisioner:
+  name: chef_zero
+
+platforms:
+<%
+    ENV['TEST_PLATFORMS'] ||= "centos7:roboxes/centos7,centos8:roboxes/centos8"
```

Comment:
I'm not sure if this is the pattern followed elsewhere, but having a default here seems a little brittle, because if you forget to set TEST_PLATFORMS in gitlab CI nothing will indicate that you made a mistake.

Reply:
This is the pattern used by the actual kitchen tests. If you forgot to set it in Gitlab CI, it will use these defaults